src/Runner.Worker/Container/ContainerHooks/HookResponse.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+namespace GitHub.Runner.Worker.Container.ContainerHooks
+{
+    public class HookResponse
+    {
+        public ResponseContext Context { get; set; }
+        public JToken State { get; set; }
+        public bool? IsAlpine { get; set; }
+    }
+
+    public class ResponseContext
+    {
+        public HookContainer Container { get; set; }
+        public IList<HookContainer> Services { get; set; }
```

Comment:
Are these going to be type`Hook Container`?

Reply:
I'd expect the hook authors to use the same container interface for their output as their input - with potentially a number of unset properties. Would you suggest we use a more specific class?

Reply:
This just maps to the [job context](https://docs.github.com/en/actions/learn-github-actions/contexts#example-contents-of-the-job-context)

We don't allow you to set anything outside of fields in that context, so I don't think there is a point in accepting them, or am I missing something?

Reply:
Will use more specific classes 👍